ui/src/app/views/workspace-edit/component.ts

```diff
@@ -80,4 +94,28 @@ export class WorkspaceEditComponent implements OnInit {
     this.workspaceCreationError = false;
     this.buttonClicked = false;
   }
+
+  updateWorkspace(): void {
+    if (!this.buttonClicked) {
```

Comment:
Could you rename `buttonClicked`? I keep losing track of what it means. `saveClicked`? `saveInProgress`?